t/page/house_wikidata.rb

```diff
@@ -19,8 +19,21 @@
     subject.title.must_equal 'EveryPolitician: Austria — Nationalrat'
   end
 
-  it 'should have popolo with wikidata' do
-    andrea = '0eedf2c9-01ea-44f4-bc6e-e5e4bf6d2add'
-    subject.popolo.persons.find_by(id: andrea).wikidata.must_equal 'Q493950'
+  it 'should pass a list of people with wikidata' do
+    res = subject.people_with_wikidata.select { |p| p[:name] == 'Cornelia Ecker' }
```

Comment:
Straggling `[:name]` rather than `.name`. This might be clearer, though, as 

```ruby
subject.people_with_wikidata.map(&:name).must_include 'Cornelia Ecker'
```